src/AccessibilityInsights.Rules/Rules.cs

```diff
@@ -44,6 +56,11 @@ public static RunResult RunRuleByID(RuleId id, IA11yElement element)
             return retVal;
         }
 
+        /// <summary>
+        /// Run all the rules in the Rules assembly.
+        /// </summary>
+        /// <param name="element"></param>
```

Comment:
>param [](start = 13, length = 5)

Should we have meaningful comments for element and returns?

Reply:
Same response as above question.